<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QWidget" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="3,10,4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lb_user_name">
       <property name="text">
        <string>用户名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_user_vip">
       <property name="text">
        <string>vip</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_q_qun">
       <property name="text">
        <string>qq群：978098280</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_chongzhi">
       <property name="text">
        <string>赞助</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_ws_con">
       <property name="text">
        <string>连接到服务器</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="toolTip">
      <string/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabWidgetPage1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>基本设置</string>
      </attribute>
      <widget class="QPushButton" name="bt_start_up">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>100</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>启动</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cb_tuanlian">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>自动团练授业</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cb_neigong">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>自动点内功</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabWidgetPage2">
      <attribute name="title">
       <string>刺探设置</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lb_log">
     <property name="text">
      <string>运行日志</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
